NSS-Day-Cohort-42/rare-server-media-moguls
# Cb register

#### Changes Made
1. Added files `request.py` and __init__.py to `users` directory.
2. Created a User class in models directory
3. added necessary get, post, put, delete methods to request_handler
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-register
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user enters information and clicks on `Register`, their data should be saved in the database for further use

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist
